src/OrchardCore.Themes/TheBlogTheme/Recipes/blog.recipe.json

```diff
@@ -11,6 +11,7 @@
 
   // The variables are evaluated the first time they are accessed, and reused across steps
   "variables": {
+    "adminUrlPrefix": "[js:configuration('OrchardCore_Admin:AdminUrlPrefix']",
```

Comment:
```suggestion
    "adminUrlPrefix": "[js: configuration('OrchardCore_Admin:AdminUrlPrefix', 'Admin']",
```

Reply:
I was thinking about that but while the `appsettings.json` has it I thought it is not needed

Reply:
I don't know how many times I have to tell you @hishamco the source on github is not what is used in a NuGet package.

Reply:
AFAIK the host app should have `appsettings.json` Am I right? regardless of the using the source or NuGet packages